src/templates/root.html

```diff
@@ -2,48 +2,85 @@
-      {% else %}
-        <a class='btn btn-secondary active' role="button" aria-pressed="true" href="/">Home</a>
-        <a class='btn btn-secondary active' role="button" aria-pressed="true" href="/server/">Servers</a>
-        <a class='btn btn-secondary active' role="button" aria-pressed="true" href="/server/create">Make a new Server</a>
-        <a class='btn btn-secondary active' role="button" aria-pressed="true" href="/register/">Sign up</a>
-        <a class='btn btn-secondary active' role="button" aria-pressed="true" href="/login/">Login</a>
-      {% endif %}
-      
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
+        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse justify-content-end" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item">
+            <a class='nav-link active' role="button" aria-pressed="true"
+              href="/support/">Support</a>
+          </li>
+          <li class="nav-item">
+            <a class='nav-link active' role="button" aria-pressed="true" href="/about/">About</a>
+          </li>
+          <li class="nav-item">
+            <a class='nav-link active' role="button" aria-pressed="true"
+              href="/servers/">Servers</a>
+          </li>
+          <li class="nav-item">
+            {% if session['logged-in'] %}
+            <li class="nav-item dropdown">
+              <a class="nav-link btn green dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                Profile
+              </a>
+              <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdownMenuLink">
+                <li><a class="dropdown-item" href="/servers/my/">Your Servers</a></li>
+                <li><a class="dropdown-item" href="/account/">Profile</a></li>
+                <li><a class="dropdown-item" href="/account/settings/">Account Settings</a></li>
+                <li><a class="dropdown-item" href="/billing/">Billing</a></li>
+                <li><hr class="dropdown-divider"></li>
+                <li><a class="dropdown-item signout" href="/logout/">Sign out</a></li>
+              </ul>
+            </li>
+            {% else %}
+            <a class='nav-link btn btn-secondary active green' role="button" aria-pressed="true" href="/register/">Get
+              Started</a>
+            {% endif %}
+          </li>
+        </ul>
+      </div>
     </div>
   </nav>
   {% endblock %}
-  <div class="m-4">
+  {% include "flash.html" %}
+  <div class="p-4">
     {% block content %} {% endblock %}
   </div>
-  </body>
-  
-  <footer>
-    {% block footer %} {% endblock %}
-  </footer>
\ No newline at end of file
+  <!-- JQuery -->
```

Comment:
Do we need this anymore, does it ever get used?

Reply:
The terminal uses it